packages/lib/src/vis/models.ts

```diff
@@ -111,3 +111,9 @@ export interface CanvasEventCallbacks {
   onPointerUp?: (evt: CanvasEvent<PointerEvent>) => void;
   onWheel?: (evt: CanvasEvent<WheelEvent>) => void;
 }
+
+export interface CustomColor {
```

Comment:
I get the impression this describes a `FallbackColor` rather than a `CustomColor`.

Reply:
Hmm not only since it tells `useCustomColor` which color property to retrieve. I hesitated with `CustomColorDef` and `COLOR_DEFS`, is that better?

Reply:
I see. Then `CustomColor` is fine